crates/ironrdp-pdu/src/gcc/network_data.rs

```diff
@@ -80,6 +80,10 @@ impl ChannelName {
     }
 
     /// Get a &str if this channel name is a valid ASCII string.
+    ///
+    /// # Panics
+    ///
+    /// If this channel name is not a valid, null-terminated ASCII string.
```

Comment:
issue: This does not panic, because it’s not possible to construct a ChannelName with an invalid string.